bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/OreGenerator.java

```diff
@@ -400,7 +405,7 @@ public void enableRotation() {
         rotation.add(this);
         for (Player p : arena.getWorld().getPlayers()) {
```

Comment:
Can we not just skip the whole loop instead of just the hologram.put method?

Reply:
Didnt quite understand what you meant

Reply:
You added the `hologram == true` check to the  if statement. But wouldnt we be able to skip the whole for loop instead?